samples/connectedhomeip/light_switch/README.rst

```diff
@@ -0,0 +1,158 @@
+
+User interface
+**************
+
+LED 1:
+    Shows the overall state of the device and its connectivity.
+    The following states are possible:
+
+    * Solid Off - The device is not paired with any light bulb device.
+    * Short Flash On (100 ms on/900 ms off) - The device is in the :ref:`pairing <chip_light_switch_sample_pairing>` phase.
+    * Solid On - The device is paired with a light bulb device.
+
+Button 1:
+    Initiates the factory reset of the device.
+
+Button 2:
+    Toggles the light on the paired light bulb device.
+
+Button 3:
+    Enables :ref:`pairing <chip_light_switch_sample_pairing>` for 10 seconds.
+    It also starts the Thread networking in the :ref:`test mode <chip_light_switch_sample_test_mode>` if the Thread network is not yet initialized.
+
+Button 4:
+    Brightens the light on the paired light bulb device.
+    The brightness level is increased gradually while the button is being pressed.
+    When the button is pressed for too long, the brightness level wraps around and becomes the lowest possible.
+
+SEGGER J-Link USB port:
+    Used for getting logs from the device or for communicating with it through the command-line interface.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/connectedhomeip/light_switch`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After building this and `chip_light_bulb_sample` samples, and programming them to your development kits, test communication between both devices by performing the following steps:
+
+.. chip_light_switch_sample_testing_start
+
+#. |connect_kit|
+   Complete this step for both kits.
+#. |connect_terminal|
+   Complete this step for both kits.
+#. On both devices, press **Button 1** to reset them to factory settings.
+#. Pair both devices by completing the following steps:
+
+   a. On the light switch device, press **Button 3** to enable :ref:`pairing <chip_light_switch_sample_pairing>` on this device.
+      The light switch becomes the Thread network Leader.
+      The following messages appear in the console for the light switch device:
+
+      .. code-block:: console
+
+         I: Device is not commissioned to a Thread network. Starting with the default configuration
+         I: Starting light bulb discovery
+
+   b. On the light bulb device, press **Button 3** to enable pairing on this device.
+      The following messages appear on the console for the light bulb device:
+
+      .. code-block:: console
+
+         I: Device is not commissioned to a Thread network. Starting with the default configuration
+         I: Started Publish service
+
+      At this point, the light bulb is discovered by the light switch device and the following messages appear on the console for the light switch device:
+
+      .. code-block:: console
+
+         I: Stopping light bulb discovery
+         I: Pairing with light bulb fe80::c051:830b:2a41:6cf1
```

Comment:
Actually it is exactly opposite, so light bulb sends message to `ff02::1`, because light switch is waiting for it :) We know that it should not be used and we are going to replace it with `ff03::1`, but as far as I know from @Damian-Nordic there is some bug in the Zephyr and currently we are not able to use that address. After fixing in Zephyr we would like to align it in our samples. Maybe we should add `TODO:` section to not forget about it?